### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by atharshaik**
added logo and remove unwanted code

### CODE REVIEW

File: data/header.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - name: "Web development" 
+      url: "/web development"
+
+
+
+  - name: "Blog"
+    url: "/blog"
+
+  - name: "Contact"
+    url: "/contact"
```
Comment (smfhashmi) [inline data/header.yml:17]: Eof is missing

File: layouts/partials/header.html [hunk 2]
```diff
@@ -1,89 +1,89 @@
...
+                                <a href="contact.html" class="btn">Get it now</a>
+                            </div>
+                        </div>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </div>
+  </header>
+  <!-- End Header Area -->
```
Comment (smfhashmi) [inline layouts/partials/header.html:89]: EOF miss

Comment (atharshaik) [reply layouts/partials/header.html:89]: done

File: data/header.yml (hunk 1 above)
Comment (atharshaik) [reply data/header.yml:17]: done

**Pull request merged**